.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -13,6 +13,12 @@ Before proceeding, please update to latest version and test if the issue persist
 ## Describe the bug in a sentence or two.
```

Comment:
Title? 
Something like Bug report for React SDK?

Reply:
Already exists